src/utils/light-attributes-mutation.js

```diff
@@ -0,0 +1,13 @@
+import assert from 'assert';
+
+export default mutation => {
+  const { name, ...patch } = mutation;
+
+  if (typeof name === 'string') {
+    assert(name.length <= 32, 'Name must be 32 characters or less');
+
+    Object.assign(patch, { name });
+  }
```

Comment:
Nitpick: `assert` could be called on `mutation.name` directly, then lines 4-5 & 8-9 wouldn't be necessary.

Reply:
Good catch 👌